### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by kristenmb**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
Adds cypress testing for UI across app.

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
Could not successfully stub so many tests are commented out as they would have manipulated the database. 

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have used tests to prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my change

### CODE REVIEW

File: cypress/integration/dashboard_spec.js [hunk 1]
```diff
@@ -4,26 +4,31 @@ describe('Dashboard', () => {
...
-  });
-
-  it('should have a Contacts Preview with three contacts', () => {
-    // cy.get('.ContactsPreview a').should('have.length', 3);
+  it('should have a Contacts Preview', () => {
+    cy.get('.ContactsPreview').find('.top-three a').should('be.visible')
+    cy.get('.ContactsPreview p').eq(0).should('contain', 'Main Contacts')
+    cy.get('.ContactsPreview').find('.all-contacts').should('contain', 'All Contacts')
+      .click().get('.Contacts').should('be.visible')
+    cy.get('.Header').find('.logo').click()
```
Comment (DrewBradley) [inline cypress/integration/dashboard_spec.js:19]: These are things it should have.

File: src/components/App.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import Dashboard from './dashboard/Dashboard'
 import Header from './header/Header'
 import Calendar from './calendar/Calendar'
 import Form from './form/Form'
-import NavBar from './navbar/NavBar'
+import NavBar from './header/navbar/NavBar'
```
Comment (DrewBradley) [inline src/components/App.js:8]: Good catch!

**Review by DrewBradley — APPROVED** [review]
Great!

**Review by richardltyler — APPROVED** [review]
Looks good!

**Review by kevinhartmann23 — APPROVED** [review]
Nice work on testing, thank you very much!

**Pull request merged**